### ISSUE TITLE
[QOL] Power Dish Signals

### ISSUE THREAD

**Issue by OmegaHaxors** [Electrical-Age/ElectricalAge#802]
As they currently exist, the power dish's signals are an absolute pain in the posterior to deal with. I am going to propose a few minor changes which will make them far more enjoyable to use:

Both dishes have a valid power node opposite of the existing one. This will allow scaling to be far easier.
Control node removed from Transmitter. Replaced with another signal node to allow further scaling.
Dishes are now controlled directly from their incoming signal, which can be fed out via signal node.
This behavior can be disabled (turning them into Control nodes) with the wrench.
All original node positions can be wrenched back to, to prevent the update breaking builds.

**Comment by AMIDIBOSS** [comment]
See: Radio tower:
https://github.com/Electrical-Age/ElectricalAge/pull/615
Is it something similar?

**Comment by OmegaHaxors** [comment]
No, this is referring to the power transmission dishes. Their nodes make it very difficult to stack them since the other nodes get in the way. For example, if you stack two dishes vertically the power node on the bottom dish needs to face down since the top dish can't send power through it.

EDIT: Your post does look freaking sweet though.
